### PR TITLE
qcow2v3: Add feature page.

### PR DESCRIPTION
**Pull request by maorlipchuk**
New pull request of qcow2v3 since the last repository encountered problems

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: @rollandf @mureinik

### CODE REVIEW

File: source/develop/release-management/features/storage/qcow2v3.html.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
+---
+title: QCOW2 compat levels
+category: feature
+authors: mlipchuk
+feature_name: QCOW2 compat levels
+feature_modules: engine,vdsm
+feature_status:  Released in oVirt 4.1
+---
+
+# QCOW2V3 Image Format
```
Comment (rollandf) [inline source/develop/release-management/features/storage/qcow2v3.html.md]: "QCOW2 compat levels" as Allon commented

File: source/develop/release-management/features/storage/qcow2v3.html.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+- Name: [Maor Lipchuk](https://github.com/maorlipchuk)
+- Email: <mlipchuk@redhat.com>
+
+
+## Detailed Description
+
+The Storage Domain's version dictated the maximum supported compat level of QCOW volumes.
+V3 Storage Domains (or older) support only 0.10 QCOW compat level, while V4 Storage Domains also support 1.1 QCOW compat level.
+For more information about features of the different QCOW versions, see [](http://wiki.qemu.org/Features/Qcow3)
```
Comment (rollandf) [inline source/develop/release-management/features/storage/qcow2v3.html.md]: You need some text between the brackets to have the link correctly

File: source/develop/release-management/features/storage/qcow2v3.html.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+  <version>
+     <major>4</major>
+     <minor>1</minor>
+  </version>
+</data_center>
+```
+
+##### Add a storage domain with storage format V4
+
+You can use explicit <storageFormat>V4</storageFormat> in the request for adding a new Storage Domain or not mention it and the default will be V4.
```
Comment (rollandf) [inline source/develop/release-management/features/storage/qcow2v3.html.md]: you need to put the "<storageFormat>V4</storageFormat>" between backtick `

File: source/develop/release-management/features/storage/qcow2v3.html.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+    <address>storage.host.address</address>
+    <path>/path/nfs</path>
+  </storage>
+  <host>
+    <name>host_name</name>
+  </host>
+</storage_domain>
+```
+
+The following is an exmaple of the response once adding a storage domain (The storage_format can be noticed):
```
Comment (rollandf) [inline source/develop/release-management/features/storage/qcow2v3.html.md]: typo exmaple -> example

File: source/develop/release-management/features/storage/qcow2v3.html.md [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+
+New QCOW volumes that will be created on a V4 Storage Domains will be created with 1.1 compatibility level.
+That also includes snapshots, so theoratically we could have an old disk with a chain of volumes that some of the volumes will be with
+compatibility level of 0.10, and others (like new snapshots) that will be with 1.1 compatibility level based on those 0.10 volumes.
+Those types of chains are supported by QEMU, and oVirt will also introduce the user the ability to upgrade any disk, so all its volumes
+will be with the same QCOW compatibility version through a REST update command which will be described later in this document.
+
+### Fetch QCOW compat for a disk through REST API
+
+Important: The disk's attrobute `QcowCompat` will only be presented for QCOW volumes only as opposed to RAW disks.
```
Comment (rollandf) [inline source/develop/release-management/features/storage/qcow2v3.html.md]: typo attrobute->attribute

**Comment by maorlipchuk** [comment]
Done for all

**Pull request merged**